src/Features/Core/Portable/NavigateTo/AbstractNavigateToSearchService.InProcess.cs

```diff
@@ -35,27 +36,27 @@ internal abstract partial class AbstractNavigateToSearchService
                 // Map our value to 'Fuzzy' as that's the lower value the platform supports.
                 (PatternMatchKind.LowercaseSubstring, NavigateToMatchKind.Fuzzy));
 
-        public static Task SearchProjectInCurrentProcessAsync(
+        public static Task SearchFullyLoadedProjectInCurrentProcessAsync(
```

Comment:
fully loaded in the context of the operation-progress-service.   effectively: if the project system has handed everything over to us, then we do the full search. otherwise, if they're still loading, we do the cached search.